tpch/tpchdata.dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM ubuntu:22.04
```

Comment:
We must run those tests in docker?

Reply:
Use a uniform env to generate data and avoid all kinds of problems. For example, It will not success to run `make` for https://github.com/databricks/tpch-dbgen.git in my macos : D

Reply:
> Use a uniform env to generate data and avoid all kinds of problems.

This is the problem `dev-setup.sh` want to address. Please consider adding them in `dev-setup.sh` so that we can reuse the docker file, image, tools...